spec/controllers/assignment_invitations_controller_spec.rb

```diff
@@ -68,6 +68,30 @@
       end
     end
 
+    context 'github repository with the same name already exists' do
+      let(:assignment_repo) { AssignmentRepo.new(assignment: assignment, user: user) }
+
+      before do
+        assignment_repo.create_github_repository
+      end
+
+      it 'creates a new assignment repo' do
+        patch :accept_invitation, id: invitation.key
+        expect(user.assignment_repos.count).to eql(1)
+      end
```

Comment:
Since you've already created the `assignment_repo` in the `let` hitting the `PATCH` route doesn't do anything. We can kill this test.